test-requirements.txt

```diff
@@ -1,3 +1,13 @@
 coverage==4.2                # https://coverage.readthedocs.io/en/coverage-4.2/changes.html
 PyVirtualDisplay==0.2        # https://github.com/ponty/PyVirtualDisplay/releases
-selenium>=2.53.6,<3.0.0      # https://github.com/SeleniumHQ/selenium/blob/master/py/CHANGES
+
+# We use Firefox and selenium for running UI tests. But using the latest 
+# versions is fragile because every few months a Firefox release is incompatible
+# with the current selenium. Since our UI tests don't depend on the latest features
+# of either we now use specific versions known to work together.
+#
+# Only update the selenium version number below if you know it works with
+# the currently-specified Firefox version.
```

Comment:
The specific version of Firefox is determined by the provisioner and is not specified in this repository. To clarify this, we may want to change this statement to something like "... if you know it works with the currently-provisioned Firefox version."